src/twisted/cred/test/test_strcred.py

```diff
@@ -264,10 +257,55 @@ def test_unixCheckerFailsPassword(self):
                            ", ".join(availability))
 
 
+class CryptTests(unittest.TestCase):
+    """
+    L{crypt} has functions for encrypting password.
+    """
+    if not crypt:
+        skip = "Required module is unavailable: crypt"
+
+    def test_verifyCryptedPassword(self):
+        """
+        L{cred_unix.verifyCryptedPassword}
+        """
+        password = "sample password ^%$"
+
+        cryptedCorrect = crypt.crypt(password, None)
```

Comment:
@rodrigc Looks like another p2/3 change.  The salt parameter isn't optional in p2.  I submitted a PR with one possible fix:
https://github.com/twisted/twisted/pull/887

Reply:
Fixed